MODULE.bazel

```diff
@@ -1,6 +1,147 @@
-###############################################################################
-# Bazel now uses Bzlmod by default to manage external dependencies.
-# Please consider migrating your external dependencies from WORKSPACE to MODULE.bazel.
-#
-# For more details, please check https://github.com/bazelbuild/bazel/issues/18958
-###############################################################################
+module(name = "rules_play_routes")
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "buildifier_prebuilt", version = "7.3.1")
+bazel_dep(name = "rules_java", version = "7.11.1")
+bazel_dep(name = "rules_jvm_external", version = "6.4")
+bazel_dep(name = "rules_pkg", version = "1.0.1")
+bazel_dep(name = "rules_scala_annex")
+
+rules_scala_annex_version = "bzlmod-migration"
+
+archive_override(
+    module_name = "rules_scala_annex",
+    integrity = "sha256-Gt+S2R53KPhfsK66bhOSXowjb35Ouf24zcT3xFZcyAs=",
+    strip_prefix = "rules_scala-{}".format(rules_scala_annex_version),
+    urls = ["https://github.com/lucidsoftware/rules_scala/archive/refs/heads/{}.zip".format(rules_scala_annex_version)],
+)
+
+bazel_dep(name = "stardoc", version = "0.7.1")
```

Comment:
Is this dep intentionally below the others or can it be moved up with the others?

Also I think this can be marked as `dev_dependency`

Reply:
I wanted to keep the `archive_override` call close to the `bazel_dep` call for `rules_scala_annex`. That way, it's not confusing where `rules_scala_annex` is coming from or what version we're using.

Also, see my comment here:
https://github.com/lucidsoftware/rules_scala/pull/74#discussion_r1873510258

Reply:
Ah got it. Makes sense.